src/lib/schemeUtils.ts

```diff
@@ -0,0 +1,209 @@
+/* eslint-disable no-bitwise */
+import {
+  Address,
+  IContractInfo,
+  ISchemeState} from "@daostack/client";
+import { rewarderContractName } from "components/Scheme/ContributionRewardExtRewarders/rewardersProps";
+import { GenericSchemeRegistry } from "genericSchemeRegistry";
+
+/**
+ * gotta load moment in order to use moment-timezone directly
+ */
+import "moment";
+import * as moment from "moment-timezone";
+
+import { getArc } from "../arc";
+
+export enum SchemePermissions {
+  None = 0,
+  IsRegistered = 1, // Always added by default in the controller
+  CanRegisterSchemes = 2,
+  CanAddRemoveGlobalConstraints = 4,
+  CanUpgradeController = 8,
+  CanCallDelegateCall = 0x10,
```

Comment:
There is also a metadata permission that I'm querying Oren about.

Reply:
sounds like this is all good for now

Reply:
I haven't seen a response from Oren about the metadata permission (that it overlaps the delegatecall permission).

Reply:
the responded and said they are the same

Reply:
Ah, got it, looks good